### PR TITLE
Minor CSS tweak for browse repository, refs #9884

### PR DESCRIPTION
**Pull request by MikeFE**
Redmine: https://projects.artefactual.com/issues/9884

### CODE REVIEW

**Comment by sevein** [comment]
Isn't this going to make the space between brick rows too big?

**Comment by sevein** [comment]
Can we update the layout/stylesheet so it's visually consistent with the advance search options toggle in the information object browser?



This is what I am seeing now with your fix: for some reason the font family is different.

**Comment by MikeFE** [comment]
How about now? :)

File: apps/qubit/modules/repository/templates/browseSuccess.php [hunk 1]
```diff
@@ -104,10 +104,8 @@
       </div>
     </div>
 
-    <div class="row">
-      <div class="span5">
-        <button id="toggleAdvancedFilters" class="btn fa fa-angle-double-down">&nbsp;<?php echo __('Advanced') ?></button>
-      </div>
+    <div id="content">
+      <a href="#" id="toggleAdvancedFilters" class="advanced-search-toggle"><?php echo __('Advanced search options') ?></a>
```
Comment (fiver-watson) [inline apps/qubit/modules/repository/templates/browseSuccess.php]: Should "advanced search options" be wrapped for i18n? do we need aria attributes here for accessibility?

Comment (MikeFE) [inline apps/qubit/modules/repository/templates/browseSuccess.php]: The __() part ensures i18n. I'll look into adding aria.

**Comment by sevein** [comment]
👍

**Comment by sevein** [comment]
@MikeFE, I ended up doing some layout and CSS clean-ups.
Please merge if you have a chance! You may still want to fix the aria stuff.

**Pull request closed without merging**